packages/system/src/font.ts

```diff
@@ -61,9 +67,46 @@ export const font = (props: FontProps): ResponsiveStyle => {
     const cssValue = props[key as FontKeys];
     if (cssValue != undefined) {
       const property = fontMappings[key as FontKeys];
-      const converter = [fontMappings.fontSize].includes(property)
-        ? toCssUnit
-        : undefined;
+
+      const userTheme = theme.getUserTheme();
+      const converter = (value: string | number): string | number => {
+        if (property === "font-family") {
+          let newValue = value;
+          if (userTheme.fonts) {
+            for (const key in userTheme.fonts) {
+              if (value === key) {
+                newValue = userTheme.fonts[key];
+                break;
+              }
+            }
+          }
```

Comment:
I think it can be refactored as follows:

```suggestion
          if (value in (userTheme.fonts ?? {})) {
            newValue = userTheme.fonts[value];
          }
```